Library/Homebrew/extend/kernel.rb

```diff
@@ -16,7 +16,7 @@ def require?(path)
 
   def ohai_title(title)
     verbose = if respond_to?(:verbose?)
-      verbose?
+      T.unsafe(self).verbose?
```

Comment:
note that `respond_to?` is not supported as a type, bc sorbet cannot validate types, arity, visibility, etc